src/api/v2/endpoint/chat.py

```
@@ -52,6 +70,22 @@ async def chat_endpoint(request: ChatRequest, db: AsyncSession = Depends(get_db)
+        return CommonResponse(
+            isSuccess=False,
+            code="AI503",
+            message="데이터베이스 연결에 실패했습니다.",
+            result=None,
+        )
+
+    except ValueError as e:
+        return CommonResponse(
+            isSuccess=False,
+            code="AI400",
+            message=f"요청 처리 중 오류 : {str(e)}",
```

Ruff (0.15.2)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)